resolution
width
height
origin_x 
origin_y
window_size
nav_msgs.OccupancyGrid probs
